apps/src/music/views/SoundsPanel2.tsx

```diff
@@ -0,0 +1,487 @@
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+import classNames from 'classnames';
+import {getBaseAssetUrl} from '../appConfig';
+import styles from './soundsPanel2.module.scss';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import MusicLibrary, {
+  SoundData,
+  SoundFolder,
+  SoundType,
+} from '../player/MusicLibrary';
+import SoundStyle from '../utils/SoundStyle';
+import FocusLock from 'react-focus-lock';
+import SegmentedButtons from '@cdo/apps/componentLibrary/segmentedButtons';
+import musicI18n from '../locale';
+
+/*
+ * Renders a UI for previewing and choosing samples. This is currently used within a
```

Comment:
Mind adding a comment around what the "2" refers to (/how this differs from v1)?

Reply:
Good call; done.